primes.rb

```diff
@@ -1,6 +1,34 @@
 class Primes
   def self.sum_to(limit = 100)
-    # TODO - add your prime number solution here...
-    "I'm working on it!"
+    nums = Array.new
+	i = 2
+	sum = 0
+
+	while i <= limit do
+      nums.push(i)
+      if (i%2 == 0 && i*2 != 4)
+        nums.delete(i)
+      end
+      if (i%3 == 0 && i*3 != 9)
+        nums.delete(i)
+      end
+      if (i%5 == 0 && i*5 != 25)
+        nums.delete(i)
+      end
+      if (i%7 == 0 && i*7 != 49)
+        nums.delete(i)
+      end
+      if (i%11 == 0 && i*11 != 121)
+        nums.delete(i)
+      end
+      i += 1
+	end
+	
+	for number in nums
```

Comment:
Using `for` and `in` isn't considered to be particularly idiomatic. A better way, since `nums` is [Enumerable](http://ruby-doc.org/core-2.2.2/Enumerable.html) would be to use something such as `reject`, `collect` or friends instead. See what you can come up with. :wink: